### ISSUE TITLE
Need to add donate page

### ISSUE THREAD

**Issue by dawnfh** [jmaster67/suicide_prev#1]
Inputting a donation page is needed. Will research ways we can input it into.

**Comment by jmaster67** [comment]
Yes, I  have a basic form i created before class, i can start with that...

**Comment by jmaster67** [comment]
I have added the donate page a link from the home page "Make a Donation"  links to my form page
